front-end/src/common/utils.ts

```diff
@@ -41,4 +55,26 @@ export function nth(n: number): string {
 
 export function getNumberWithSuffix(n: number): string {
   return `${n}${nth(n)}`
-}
\ No newline at end of file
+}
+
+export const StatGenerators = {
+  'KDA': new KDAStatGenerator(),
+  'DMG_PERCENT': new DMGPercentStatGenerator(),
+  'CSPM': new CSPMStatGenerator(),
+  'DPM': new DPMStatGenerator(),
+  'GOLD_SHARE': new GoldShareStatGenerator(),
+  'KP_PERCENT': new KPPercentStatGenerator(),
+  'AVERAGE_VS': new AverageVisionScoreStatGenerator(),
+  'DEATH_PERCENT': new DeathPercentStatGenerator(),
+  'DMG_PER_GOLD': new DPGStatGenerator(),
+  'VS_PERCENT': new VisionScorePercentStatGenerator(),
+  'GPM': new GPMStatGenerator(),
+  'SOLO_KILL': new SoloKillStatGenerator(),
+  'DMG_TAKEN_PM': new DamageTakenPerMinuteStatGenerator(),
+  'XP_DIFF_15': new DiffStatGenerator(DiffEnum.XP, 15),
+  'XP_DIFF_25': new DiffStatGenerator(DiffEnum.XP,25),
+  'GOLD_DIFF_15': new DiffStatGenerator(DiffEnum.GOLD, 15),
+  'GOLD_DIFF_25': new DiffStatGenerator(DiffEnum.GOLD,25),
+  'CS_DIFF_15': new DiffStatGenerator(DiffEnum.CS, 15),
+  'CS_DIFF_25': new DiffStatGenerator(DiffEnum.CS,25),
```

Comment:
Good change

Reply:
![gladge](https://preview.redd.it/9mpl8964nuc51.png?auto=webp&s=9094cf8195cd771ab5f7caaea8f7799f55ca122b)